### PR TITLE
Alignment of displayed user data and content

### PR DESCRIPTION
**Pull request by Priyaraj17**
## Summary

I have removed text-align: right/left from the css styles and added dir = "auto" to their HTML. 

## References

Issue: #3781

## Reviewer guidance

…

----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/assets/src/views/AppBar.vue [hunk 1]
```diff
@@ -263,7 +263,7 @@
     position: fixed;
     right: 8px;
     z-index: 8;
-
+    outline: none;
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/AppBar.vue:266]: Why was this added? I see it in a number of places

File: kolibri/core/assets/src/views/AppError/ReportErrorModal.vue [hunk 2]
```diff
@@ -95,11 +96,3 @@
 
 </script>
 
-
-<style lang="scss" scoped>
-
-  .error-detail-modal {
-    text-align: left;
-  }
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/AppError/ReportErrorModal.vue:103]: if you're removing an entire class, you should remove the reference to it in the `template` also

File: kolibri/core/assets/src/views/PaginatedListContainer.vue [hunk 3]
```diff
@@ -150,7 +150,7 @@
 
   .pagination-nav {
     margin-bottom: 8px;
-    text-align: right;
+
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/PaginatedListContainer.vue:153]: I believe replacing `text-align: right` with `dir="auto"` will not have the intended behavior

**Review by indirectlylit — COMMENTED** [review]
Hi @Priyaraj17 thank you for your effort. This is a somewhat tricky issue.

Have you tested each of these changes with both LTR and RTL to be sure things are working as intended? I see a lot of changes here which don't quite make sense to me because they are affecting [layout alignment](https://github.com/learningequality/kolibri/issues/3781#issuecomment-814326599), not text direction.

Typically we ask that submitted PRs come with before and after screenshots. I understand there are many here, plus seeing it in both RTL and LTR languages, so I don't think you'll need to provide screenshots of every change. However please provide at least a few to show the intended effect of your work.

File: kolibri/core/assets/src/views/AppBar.vue (hunk 1 above)
Comment (Priyaraj17) [reply kolibri/core/assets/src/views/AppBar.vue:266]: Hey, actually I was working on two different issues simultaneously. These changes are of a different issue.
I believe this issue requires more research. Can I put this issue on hold and take some time understanding it?

Comment (rtibbles) [reply kolibri/core/assets/src/views/AppBar.vue:266]: When you initially suggested this solution to the issue, @jonboiser pointed out that this absolutely was not the approach to take: https://github.com/learningequality/kolibri/issues/7974#issuecomment-815955969

Subsequently you have opened three separate PRs with these changes. Please pay attention to the feedback from repository maintainers so that we do not spend our time reviewing code that you have already been told is not an appropriate fix for the issue at hand.

Comment (Priyaraj17) [reply kolibri/core/assets/src/views/AppBar.vue:266]: @rtibbles Actually I was having some difficulties. These PRs were created by mistake, I was having difficulty while running the server. I am sorry for this. I will ask for their feedback again and try to understand more. Thank you. 
Shall I close this pull request or How should I proceed further?

**Comment by indirectlylit** [comment]
>  Shall I close this pull request or How should I proceed further?

Yes, if you'd like to work on this please:

1. start from a fresh branch so there are no unrelated changes in your PR
2. make sure you understand our [developer guidance on text alignment](https://kolibri-dev.readthedocs.io/en/develop/i18n.html#text-alignment)
3. test each change you make with both LTR and RTL languages, and include screenshots of the before and after behaviors

**Comment by Priyaraj17** [comment]
Thank you so much for this. I will be working on it and test all the changes according to guidelines.
